### ISSUE TITLE
Also add Firefox's color contrast checker for text on background image

### ISSUE THREAD

**Issue by mgifford** [microsoft/accessibility-insights-web#5452]
Firefox has this now, it is an issue on Axe-core but also should be something that is included in Accessibility Insights
https://github.com/dequelabs/axe-core/issues/3447

Gradients are common and need to be easier to evaluate. Firefox does it this way:
![Firefox screenshot](https://user-images.githubusercontent.com/116832/166488095-f362c9cd-c723-4ed3-81f4-d52298d02617.png)

It should be something that we don't need to manually evaluate (or perhaps less-so).

**Comment by ferBonnin** [comment]
After triaging with the team, this is not something we think we can do as an extension.
